src/test/scala/mesosphere/marathon/core/task/update/impl/TaskStatusUpdateProcessorImplTest.scala

```diff
@@ -294,6 +293,26 @@ class TaskStatusUpdateProcessorImplTest extends AkkaUnitTest {
       Then("not do anything else")
       verifyNoMoreInteractions()
     }
+
+    "receiving an running update for known instnace but unknown task" in new Fixture {
```

Comment:
1. Nit: typo.
2. Nit: The test start with ` "The TaskStatusUpdateProcessor implementation" should`. Scalatest is converting it to names such as `The TaskStatusUpdateProcessor implementation should receiving an running update for known instance but unknown task`.  I believe it should be `The TaskStatusUpdateProcessor implementation should not kill unknown tasks but ignore them`.